backend/pipeline/filter/legacy.go

```diff
@@ -77,12 +77,12 @@ func (l *LegacyAdapter) Filter(ctx context.Context, ref *corev2.ResourceReferenc
 	filter, err := l.Store.GetEventFilterByName(tctx, ref.Name)
 	cancel()
 	if err != nil {
-		logger.WithFields(fields).WithError(err).Warning(getFilterErr.Error())
+		logger.WithFields(fields).WithError(err).Warning(errCouldNotRetrieveFilter.Error())
 		return false, err
 	}
 	if filter == nil {
-		logger.WithFields(fields).WithError(err).Warning(getFilterErr.Error())
-		return false, fmt.Errorf(getFilterErr.Error())
+		logger.WithFields(fields).WithError(err).Warning(errCouldNotRetrieveFilter.Error())
+		return false, fmt.Errorf(errCouldNotRetrieveFilter.Error())
```

Comment:
Nitpick: could we just return `errCouldNotRetrieveFilter` instead of creating another error?